ci-operator/config/codeready-toolchain/workload-analyzer/codeready-toolchain-workload-analyzer-master.yaml

```diff
@@ -0,0 +1,44 @@
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+releases:
+  initial:
+    integration:
+      name: "4.17"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.17"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: "3"
+      memory: 250Mi
+tests:
+- as: e2e
+  steps:
+    cluster_profile: devsandboxci-aws
+    env:
+      BASE_DOMAIN: devsandboxci.devcluster.openshift.com
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        export WEBHOOK_URL=$(cat /usr/local/sandbox/ci-secrets/webhook_url)
+        make test-e2e
```

Comment:
correct me if I'm wrong, but the `test-e2e` target will try to build the images of WA and alert-manager, but is this something we can do in the `tests`? shouldn't be there a step/job that would build the images first? Keep in mind that we don't do that in the other repos because we build the images in GH actions (because of the complicated setup of the operators).

Reply:
`test-e2e` will build the operators, wa, and alert-manager. WA and alert-manager are built through image registry.

Reply:
yes, but are you sure that we can do it in this step/job? can we run `podman build` in the test container? I'm asking because it wasn't possible previously, the images to be built had to be defined in a separate section.

Reply:
https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images (but we don't need to promote them) 
IIRC, openshift-ci builds the images and pushes them to their internal openshift-ci registry that is available from the cluster we are running the e2e-tests with. But things may have changed since I've seen it for the last time.

Reply:
> test-e2e will build the operators

just to be precise, it doesn't build the operators, it only installs them from the public quay registry. The only images we need to build are WA and alert-manager.

Reply:
Understood! Tysm! Working on added a separate step to build the images